### PR TITLE
Add subtitution vars

### PR DESCRIPTION
**Pull request by miguelpuiggarcia**

### CODE REVIEW

**Review by dbartalos — APPROVED** [review]
🚀

File: gcp/cloud-run-v2/variables.tf [hunk 1]
```diff
@@ -257,6 +257,12 @@ variable "cloud_armor" {
   }
 }
 
+variable "substitution_trigger" {
```
Comment (ducc) [inline gcp/cloud-run-v2/variables.tf:260]: `trigger_substitutions`?

Comment (miguelpuiggarcia) [reply gcp/cloud-run-v2/variables.tf:260]: yes sir :D

Comment (miguelpuiggarcia) [reply gcp/cloud-run-v2/variables.tf]: done

**Review by ducc — APPROVED** [review]

**Pull request merged**